nicsdru_nidirect_theme.theme

```diff
@@ -13,12 +13,18 @@ use Drupal\user\Entity\User;
  */
 function nicsdru_nidirect_theme_preprocess_page(array &$variables) {
   if ($node = \Drupal::routeMatch()->getParameter('node')) {
-    if ($node->bundle() == 'article') {
+    if ($node->bundle() == ('article' || 'application' || 'contact'
```

Comment:
As this appears to be applying styling to bundles that fall under the idea of 'articles', I'd be tempted to create this list as an array such as $article_bundle_types = ['article', 'application'] etc and do a check if the bundle() is in_array(). If there are gonna be a few distinctions like this it might be a good idea to declare these as a constants at the very top of the file, or as an include so developers can easily keep track.

Reply:
I'll change this into an array. Might need to refactor when I figure out if there are implications for other content type layouts